### PR TITLE
Add sequence insert support to OracleHook

### LINKED ISSUE #42494: Enter sequence values when using Oracle hooks
### Description

If a sequence column exists when using Oracle Hook, there is no condition to enter the corresponding value.
Then, when the sequence is hung in the ID value, an error called ID cannot insert null into ~

### Use case/motivation

When using functions such as bulk_insert_rows, it would be nice to add a parameter that can put conditions corresponding to the sequence

### Related issues

_No response_

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

### PR DESCRIPTION
**Pull request by Lee2532**
If Oracle had a sequence column, it could not be entered with existing logic. We received an additional sequence column and sequence name in the existing function so that you could enter it

Closes: #42494

### CODE REVIEW

**Comment by Lee2532** [comment]
@potiuk 
hi, Can you check it again?

**Comment by Lee2532** [comment]
@potiuk 

Hello
This is the PR I wrote again because I sent the wrong PR last time. Can you check it? Thank you (related PR : https://github.com/apache/airflow/pull/42778)

File: providers/src/airflow/providers/oracle/hooks/oracle.py [hunk 1]
```diff
@@ -350,11 +354,23 @@ def bulk_insert_rows(
             self.set_autocommit(conn, False)
         cursor = conn.cursor()  # type: ignore[attr-defined]
         values_base = target_fields or rows[0]
-        prepared_stm = "insert into {tablename} {columns} values ({values})".format(
-            tablename=table,
-            columns="({})".format(", ".join(target_fields)) if target_fields else "",
-            values=", ".join(f":{i}" for i in range(1, len(values_base) + 1)),
-        )
+
+        if sequence_column and sequence_name:
```
Comment (potiuk) [inline providers/src/airflow/providers/oracle/hooks/oracle.py:363]: What happens when only sequence_column but no seuence_name (or the other way round) are specified? I think it should be treated as error. You either need none or both.

Comment (Lee2532) [reply providers/src/airflow/providers/oracle/hooks/oracle.py:363]: That's right.
Currently, an error occurs even if you do not enter either of them.
I'll add a code for that case. Thank you

File: providers/src/airflow/providers/oracle/hooks/oracle.py [hunk 2]
```diff
@@ -350,11 +354,28 @@ def bulk_insert_rows(
             self.set_autocommit(conn, False)
         cursor = conn.cursor()  # type: ignore[attr-defined]
         values_base = target_fields or rows[0]
-        prepared_stm = "insert into {tablename} {columns} values ({values})".format(
-            tablename=table,
-            columns="({})".format(", ".join(target_fields)) if target_fields else "",
-            values=", ".join(f":{i}" for i in range(1, len(values_base) + 1)),
-        )
+
+        if (sequence_column is None) != (sequence_name is None):
```
Comment (potiuk) [inline providers/src/airflow/providers/oracle/hooks/oracle.py]:
```suggestion
        if bool(sequence_column) ^ bool(sequence_name):
```

Comment (potiuk) [reply providers/src/airflow/providers/oracle/hooks/oracle.py]: More pythonic and better readable IMHO

Comment (Lee2532) [reply providers/src/airflow/providers/oracle/hooks/oracle.py]: I agree

**Review by potiuk — APPROVED** [review]

**Pull request merged**